automation/components-functests.setup.sh

```diff
@@ -0,0 +1,54 @@
+#! /bin/bash
+set -exu
+
+# Create an ephemeral cluster with latest changes
+# CNAO and clones the desired component repository
+#
+# This script exports:
+# - KUBECONFIG
+#   kubeconfig full path so you can use`kubectl` binary or
+#   CNAO's `./cluster/kubectl.sh` script directly.
+#
+# - TMP_COMPONENT_PATH
+#   The desired component temp directory
+#
+# Example:
+# export COMPONENT="KubeMacPool"
+# source automation/e2e-functest.setup.sh
```

Comment:
Again, COMPONENT can be set next to `source` to limit the scope

Reply:
Done